app/src/main/java/com/example/rezepteapp/model/RecipeModel.java

```diff
@@ -0,0 +1,5 @@
+package com.example.rezepteapp.model;
+
+public class RecipeModel {
+    private Recipe recipe;
```

Comment:
Mir fehlen da noch die Methoden. Was passiert, wenn jemand auf ein Rezept drückt? Dann muss ja das passende Rezept aus der Datenbank/Liste gesucht werden und dann an die view zurückgegeben werden. Ich würde in den Models den Ort/Zeitpunkt sehen, dass da die methoden, die (indirekt) die Datenbank oder REST Abfrage machen, aufgerufen werden (kann auch erstmal mit statischem Inhalt oder Kommentaren passieren)

Reply:
Alles klar, wird umgesetzt 👍

Reply:
@Resi-r Ich habe die Methoden, die mit einem Datenbankaufruf in Verbindung stehen, eingebaut (Logik muss noch folgen)

In dem RecipeDbOpenHelper sind bissher nur methoden zum einfügen/updaten, ich würde dort ganz gerne Methoden für die Abfrage von Daten einbauen, außer du hast schon etwas in dieser richtung gemacht.